### PR TITLE
tentative fix for subset for tomo and subtomo

### PR DESCRIPTION
**Pull request by adrianq**
Re https://github.com/I2PC/scipion/issues/2036, @pconesa and I implemented this quick fix that seems to fix the problem. Thoughts @delarosatrevin? Too hacky?

### CODE REVIEW

**Comment by delarosatrevin** [comment]
Hi @adrianq and @pconesa, 
Yes, it seems quite hacky (even playing with fire) for me, anyway, I willing to accept it somehow due to the lack of time to discuss and implement a better solution for it in the near future. 

Have you checked that with these changes the subset is created? Are changes in the ShowJ java code required?

Last, but not least, I think the settatrr lines should not be where they are now. The package 'tomo' is the top level one...and it contains sub-packages 'objects' and 'protocols' so, in the 'tomo' __init__.py is better not to add dependencies of sub-packages, since it can create cyclic dependencies for imports and definitions. Maybe tomo.protocols is a better place.

**Comment by pconesa** [comment]
Thanks @delarosatrevin , I totally agree that is quite hacky, no doubt! But yes it will work with current Scipipn 2.0 released without any changes in the ShowJ (which I was surprised how it can cope with new classes) 

About the place to place the code, if it works adding it in the objects subpackage, to me, it seems a better place?

**Comment by delarosatrevin** [comment]
I think in most plugins, the 'protocols' subpackage depends on the 'objects' , no the other way around...that's why I think it is better to move the code to tomo.protocols, since you are importing both objects and protocols.

**Comment by pconesa** [comment]
I'm ok with tomo.protocols then.

**Comment by adrianq** [comment]
@delarosatrevin @pconesa  Done! I have just migrated that bit of code to the other tomo.protocols class

File: tomo/protocols/__init__.py [hunk 1]
```diff
@@ -37,3 +39,10 @@
 from .move_to_plugins.protocol_ts_gctf import ProtTsGctf
 from .move_to_plugins.protocol_imod_auto3d import ProtImodAuto3D
 from .move_to_plugins.protocol_imod_etomo import ProtImodEtomo
+
+
```
Comment (delarosatrevin) [inline tomo/protocols/__init__.py]: Please add a descriptive comment about the reasons of this hack and possible future solutions.

Comment (adrianq) [reply tomo/protocols/__init__.py]: Sure I will add it here https://github.com/scipion-em/scipion-em-tomo/pull/15 which is the PR that will make it into the tomo branch

**Pull request merged**